tests/generators/kzg_4844/main.py

```diff
@@ -509,6 +523,21 @@ def case06_verify_blob_kzg_proof_batch():
         'output': None
     }
 
+    # Edge case: Incorrect `G1_POINT_AT_INFINITY` Proof
+    blob = BLOB_RANDOM_VALID1
+    commitment = spec.blob_to_kzg_commitment(blob)
+    proof = spec.G1_POINT_AT_INFINITY
+    assert not spec.verify_blob_kzg_proof_batch([blob], [commitment], [proof])
+
+    yield 'verify_blob_kzg_proof_batch_case_incorrect_proof_point_at_infinity', {
```

Comment:
Nit: I think it looks more consistent without the blank line:
```suggestion
    assert not spec.verify_blob_kzg_proof_batch([blob], [commitment], [proof])
    yield 'verify_blob_kzg_proof_batch_case_incorrect_proof_point_at_infinity', {
```